app/views/podcast_switcher/_dropdown.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="prx-podcast-switcher dropdown" data-controller="focus">
 
-  <button class="btn btn-white dropdown-toggle" type="button" id="podcast-switcher" data-bs-toggle="dropdown" aria-expanded="false" data-action="focus#focus">
+  <button class="btn btn-white dropdown-toggle" type="button" id="podcast-switcher" data-bs-toggle="dropdown" aria-expanded="false" data-action="focus#focus" data-controller="hotkey">
```

Comment:
Brought back the switcher for podcasts#index.  So I can just type `/` and open it!